packages/shared/src/modules/quiz/libs/types/quiz-answers-request-dto.type.ts

```diff
@@ -1,5 +1,6 @@
 type QuizAnswersRequestDto = {
 	answerIds: number[];
+	categoryIds?: number[] | undefined;
```

Comment:
```suggestion
	categoryIds?: number[];
```

Reply:
If we want this to work without undefined in type union, we'll have to do this (or something along the lines):
<img width="1440" alt="Screenshot 2024-09-18 at 14 36 12" src="https://github.com/user-attachments/assets/4cbe388d-f5b9-4e19-a6d0-ce2b759a1f9f">
This is done in order to avoid the error in the terminal on the screenshot above